include/proxy-wasm/wasm_vm.h

```diff
@@ -29,58 +29,87 @@ namespace proxy_wasm {
 
 class ContextBase;
 
-// These are templates and its helper for constructing signatures of functions calling into and out
-// of WASM VMs.
-// - WasmFuncTypeHelper is a helper for WasmFuncType and shouldn't be used anywhere else than
+// These are templates and its helper for constructing signatures of functions calling into Wasm
+// VMs.
+// - WasmCallInFuncTypeHelper is a helper for WasmFuncType and shouldn't be used anywhere else than
```

Comment:
Nit: the direction in `CallIn` isn't very obvious, could you use `HostCall` instead?

Reply:
well this is about calling into Wasm VMs, not host calls from Wasm VMs to the hosts.. but yeah I agree this isn't very obvious.. any idea?

Reply:
Ah, I saw the existing `WasmCallback...` templates, so I assumed this is for hostcalls, but it looks that the host implementation uses "callbacks" for hostcalls, and "functions" for in-VM callbacks... although, even that is inconsistent and we use "host function" in some runtime implementations.

It's probably too big of a cleanup to do it as part of this PR, so let's ignore it for now.